py-rattler/docs/index.md

```diff
@@ -16,6 +16,17 @@ Rattler is written in Rust and tries to provide a clean API to its functionaliti
 With the primary goal in mind we aim to provide bindings to different languages to make it easy to integrate Rattler in non-rust projects.
 Py-rattler is the python bindings for rattler.
 
+## Quick-Start
+
+Let's see an example to learn some of the functionality the library has to offer.
+
+```python
+--8<-- "examples\solve_and_install.py"
+```
+
+Py-rattler provides friendly high level functions to download dependencies and create environments.
+This is done through for instance the `solve` and `install` functions.
```

Comment:
there is a bit of word doubling going on here ("done through for instance")

Reply:
```suggestion
The `solve` and `install` functions are excellent examples of such high-level functions.
```
Something like this?